arviz/plots/ecdfplot.py

```diff
@@ -10,14 +12,16 @@
 def plot_ecdf(
     values,
     values2=None,
+    eval_points=None,
```

Comment:
Is there a name we use elsewhere in the codebase that we should instead use here for consistency?